third_party/pyth/price-service/src/listen.ts

```diff
@@ -27,7 +27,9 @@ export type PriceInfo = {
 };
 
 export interface PriceFeedPriceInfo {
-  getLatestPriceInfo(priceFeedId: string): PriceInfo | undefined;
+  getPriceIds(): Set<HexString>;
```

Comment:
nitpick: the name `PriceFeedPriceInfo` is not great. this interface is basically the API for the spy right? maybe something like SpyApi would be better?